File: lista simple/sistema-contactos/src/Nodo.java
public class Nodo {
    private contaco Dato ;

    public contaco getDato() {
        return Dato;
    }

    public void setDato(contaco dato) {
        this.Dato = dato;
    }

    public Nodo getSiguiente() {
        return siguiente;
    }

    public void setSiguiente(Nodo siguiente) {
        this.siguiente = siguiente;
    }

    Nodo siguiente;
    
    
    

    public Nodo(contaco dato) {
        this.dato = dato;
    }

    public Object setDato() {
    }
}
}

File: lista simple/Ranking_de_juegos/src/Nodo.js
class Nodo {
    constructor(dato) {
        this.dato = dato;
        this.siguiente = null;
    }

    getDato() {
        return this.dato;
    }

    setDato(dato) {
        this.dato = dato;
    }

    getSiguiente() {
        return this.siguiente;
    }

    setSiguiente(siguiente) {
        this.siguiente = siguiente;
    }
}